roles/homer/templates/config.yml.j2

```diff
@@ -52,97 +52,141 @@ links:
 services:
   - name: "Deployment"
     icon: "fas fa-wrench"
+    {% if homer_url_ara or homer_url_flower or homer_url_netbox or homer_url_vault %}
     items:
+      {% if homer_url_ara is defined and homer_url_ara|length %}
```

Comment:
Since the URL variables are all set by default, I think it is safe to assume that they are defined, so it should be enough just to check the length. Likely an explicit "length > 0" check might be safer, though.